zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -98,7 +98,7 @@ fn create_market_common<T: Config>(
         scoring_rule,
     }
     .dispatch_bypass_filter(RawOrigin::Signed(caller.clone()).into())?;
-    let market_id = T::MarketCommons::latest_market_id()?;
+    let market_id = <zrml_market_commons::Pallet<T> as MarketCommonsPalletApi>::latest_market_id()?;
```

Comment:
This is inconsistent use of syntax. You use Chris' suggestion in `prediction-markets`, but my (bad) suggestion here.

Reply:
oh missed this file.